### PR TITLE
CORCI-1103 Build: Add CentOS 8.4 and Leap 15.3

### PR DESCRIPTION
**Pull request by JohnMalmberg**
Signed-off-by: John E Malmberg <john.e.malmberg@intel.com>

### CODE REVIEW

File: packaging/Makefile_packaging.mk [hunk 1]
```diff
@@ -125,8 +130,7 @@ all: $(TARGETS)
 	xz -z $<
 
 _topdir/SOURCES/%: % | _topdir/SOURCES/
-	rm -f $@
-	ln $< $@
+	if [ ! -d "$@" ]; then rm -f "$@"; ln "$<" "$@"; fi
```
Comment (frostedcmos) [inline packaging/Makefile_packaging.mk:133]: doesn't $@ expand to all args? also what is $< ? online guides dont seem to specify that one

Comment (jolivier23) [inline packaging/Makefile_packaging.mk:133]: What is $<

Also, isn't $@ a multiple argument thing.   Should this not be a for loop or soemthing

for x in $@; do
# do stuff on $x
done

Comment (JohnMalmberg) [reply packaging/Makefile_packaging.mk:133]: $< is the first dependency in the list of dependencies.   It is replaced by make before Bash sees it.
$@ is the effective target of the make rule.  It is replaced by make before Bash sees it.
For some unknown reason the recipe for this rule is being logged as run after a successful make build with the "%" apparently replaced by an empty string, which causes the "rm" command to emit a diagnostic.
I have not worked out why that is happening, but this is an attempt to suppress the diagnostic from the logs.

Comment (JohnMalmberg) [reply packaging/Makefile_packaging.mk:133]: "$@" and "$<" are makefile variables, bash does not see them.

**Review by jolivier23 — APPROVED** [review]

**Review by jolivier23 — APPROVED** [review]

**Comment by JohnMalmberg** [comment]
Obsolete

**Pull request closed without merging**